File: M5.Pt2/menoramajor.py
# encoding: utf-8
# Programa que mostra els nombres introduïts de menor valor a major
primer = int(input("Escriu el primer número: "))
segon = int(input("Escriu el segon número: "))
tercer = int(input("Escriu el tercer número: "))
if primer > segon and primer > tercer:
	if segon > tercer:
		print tercer,segon,primer
	else:
		print segon,tercer,primer
elif segon > primer and segon > tercer:
	if tercer > primer:
		print primer,tercer,segon
	else:
		print tercer,primer,segon
if tercer > segon and tercer > primer:
	if segon > primer:
		print primer,segon,tercer
	else:
		print segon,primer,tercer

File: M5.Pt2/MENORAMAJOR.java
import java.io.*;

/**
 * Introdueix 2 dades i mostra les dades de menor a major
 **/
public class MENORAMAJOR {
  public static void main(String[] args) throws IOException {
    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
    String inputNumA;
    String inputNumB;
    int numA;
    int numB;
    System.out.print("Introdueix el primer número: ");
    inputNumA = reader.readLine();
    numA = Integer.parseInt(inputNumA);
    System.out.print("Introdueix el segon número: ");
    inputNumB = reader.readLine();
    numB = Integer.parseInt(inputNumB);
    if (numA > numB){
		System.out.println(numB+" "+numA);
	}
	else{ 
		System.out.println(numA+" "+ numB);
	}
}
}
